Backend/mentii/user_ctrl.py

```diff
@@ -345,6 +346,21 @@ def joinClass(jsonData, dynamoDBInstance, email=None):
         response.addToPayload('code', updatedClass['code'])
   return response
 
+def leaveClass(jsonData, dynamoDBInstance, email=None):
```

Comment:
It literally calls that function. It is a wrapper function for it because we need more handling in this case.